CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 1.0.2
   - Features Added
+    - First loading speed improvement
```

Comment:
Please remove it from here, 1.0.2 has already been released. This fix will go to the next release.

Reply:
Fixed. Added new section

Reply:
@roienatan anything else?

Reply:
Let's wait for @orenyodfat review.